core/exception/arguments_spec.rb

```diff
@@ -1,11 +0,0 @@
-require_relative '../../spec_helper'
-
-describe "ArgumentError" do
-  it "is a subclass of StandardError" do
-    StandardError.should be_ancestor_of(ArgumentError)
-  end
-
-  it "gives its own class name as message if it has no message" do
-    ArgumentError.new.message.should == "ArgumentError"
-  end
```

Comment:
ArgumentError is the only Exception for which this is tested.  All, or none?

Reply:
Yeah, this is likely already tested in Exception#message specs.